spk/shairport-sync/Makefile

```diff
@@ -0,0 +1,34 @@
+SPK_NAME = shairport-sync
+SPK_VERS = 2.2.3
+SPK_REV = 1
+SPK_ICON = src/airplay.png
```

Comment:
We're probably going to need a different logo here, because the current one is owned/copyrighted by Apple as far as I can tell.